src/main.js

```diff
@@ -182,6 +183,24 @@ function backToMainPage(){
   makePosterForm.classList.add('hidden');
   savedPostersPage.classList.add('hidden');
 }
+
+function captureFormValuesOnClick() {
```

Comment:
this one looks really good! I think we could combine the input lines together later when we refactor